packages/xml-toolkit/lib/extension.js

```diff
@@ -16,21 +17,37 @@ function activate() {
     },
   };
 
+  const meta = require(resolve(__dirname, "..", "package.json"));
```

Comment:
You should make sure this works with the webpack packaging (e.g. `__dirname` might be problematic, or you might not be in the folder you expect)

Reply:
yeah thanks I discovered issues with this while testing and fixed them (in upcoming change)